target/linux/generic/hack-6.6/650-netfilter-add-xt_FLOWOFFLOAD-target.patch

```diff
@@ -560,8 +560,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	if (!flow)
 +		goto err_flow_alloc;
 +
-+	if (flow_offload_route_init(flow, &route) < 0)
-+		goto err_flow_add;
++	flow_offload_route_init(flow, &route);
```

Comment:
This needs some more changes, see https://github.com/openwrt/openwrt/pull/14766/files#diff-30f3b778748efee237bbf599e600ea8d7558fac5806377db5838352b39744e77

Reply:
OK